its/src/test/java/com/sonar/it/csharp/Tests.java

```diff
@@ -94,10 +94,12 @@ public static void runMSBuild(Orchestrator orch, Path projectDir, String... argu
   }
 
   private static Path getMsBuildPath(Orchestrator orch) {
-    String msBuildPathStr = orch.getConfiguration().getString(MSBUILD_PATH, "C:\\Program Files (x86)\\MSBuild\\14.0\\bin\\MSBuild.exe");
+    String msBuildPathStr = orch.getConfiguration().getString(MSBUILD_PATH, "C:\\Program Files (x86)\\Microsoft "
```

Comment:
Break line at `\\`, otherwise it's very easy to miss the space at the end of the line

Reply:
Automatic break from IntelliJ... I didn't even noticed it.